package/gluon-web-network/luasrc/lib/gluon/config-mode/model/admin/network.lua

```diff
@@ -102,15 +102,28 @@ uci:foreach("system", "gpio_switch", function(si)
 		end
 
 		local texts = {
-			["^PoE Power Port(%d*)$"] = function(m) return translatef("Enable PoE Power Port %s", m[1]) end,
-			["^PoE Passthrough$"] = function() return translate("Enable PoE Passthrough") end,
+			["^PoE Power Port(%d*)$"] = {
+				["func"] = function(m) return translatef("Enable PoE Power Port %s", m[1]) end,
+				["active_low"] = false,
```

Comment:
How about the following naming? (and not using the `[""]` syntax without reason)
```suggestion
				get_title = function(m) return translatef("Enable PoE Power Port %s", m[1]) end,
				active_low = false,
```

Reply:
Works for me